deploy/basic/README_MSZ.md

```diff
@@ -21,10 +21,37 @@ If using local Terraform or on CloudShell:
 - Run `terraform init` to init the terraform providers
 - Run `terraform apply` to create the resources on OCI
 
-You can also include Web Application Firewall (WAF), DNS or other components as desired:
+You can also include Web Application Firewall (WAF), DNS or other components as desired.  Note that a public certificate will be required for WAF, and thus requires a DNS zone be created.  See below for more specific instructions.
 
 ![MuShop Basic Infra MSZ Demo](../../images/basic/00-Topology-v1.2.0.MSZ-demo.svg)
 
+### Enabling DNS, Certificates and WAF 
```

Comment:
Should mention that these steps are optional

Reply:
Will fix this and make it more obvious